cpp/src/rolling/rolling.cu

```diff
@@ -131,6 +134,37 @@ void gpu_rolling(column_device_view input,
   }
 }
 
+template <typename T, typename agg_op, rolling_operator op, int block_size, bool has_nulls,
+          typename WindowIterator>
+__device__
+void kernel_gpu_rolling(specialization_type<dictionary32_tag>,
+                 column_device_view input,
+                 mutable_column_device_view output,
+                 size_type * __restrict__ output_valid_count,
+                 WindowIterator preceding_window_begin,
+                 WindowIterator following_window_begin,
+                 size_type min_periods)
+{
+  release_assert(false && "dictionary type not supported");
+}
+
+
+template <typename T, typename agg_op, rolling_operator op, int block_size, bool has_nulls,
+          typename WindowIterator>
+__launch_bounds__(block_size)
+__global__
+void gpu_rolling(column_device_view input,
```

Comment:
Potential stepping on toes happening with @rgsl888prabhu's #3786

Reply:
Thanks Devavret. This is actually in #3577. I just undid this change there but have not had a chance to merge the PR with this one yet.